src/app/app.models.ts

```diff
@@ -1,30 +1,22 @@
-export interface IWatch {
+export interface IItem {
     id: number;
-
     image: string;
-
     name: string;
-
+    price: number;
     description: string;
+}
 
+export interface IWatch extends IItem {
```

Comment:
create separate interface IWatchDetails that will contain all listed below fields only (without extending IItem). In turn current interface will extend IWatchDetail and IItem.